### ISSUE TITLE
[API] Localization - List languages

### ISSUE THREAD

**Issue by TorstenDittmann** [appwrite/appwrite#351]
We could add another API endpoint to list languages including their code according to https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

This might be a good addition to the `locale header` functionality.

Endpoint: **GET** `/locale/languages`
Response: 
```
[
...
{
    name: "German",
    nameNative: "Deutsch",
    code: "de"
},
...
]
```

If you consider adding this, I can work on that and can re-animate my PHP 5 skills 😄

**Comment by eldadfux** [comment]
Sounds cool. Let's do it.

Excited to see your first feature :) we'll upgrade you on Discord to be a PHP8 ninja...🥋

**Comment by eldadfux** [comment]
Old time :) This is now available as part of v0.7 ✅
